free-programming-books-zh.md

```diff
@@ -292,7 +292,7 @@
 * [Joel谈软件](https://web.archive.org/web/20170616013024/http://local.joelonsoftware.com/wiki/Chinese_(Simplified))
 * [selenium 中文文档](https://github.com/fool2fish/selenium-doc)
 * [开源软件架构](http://www.ituring.com.cn/book/1143)
-* [持续集成（第二版）](http://article.yeeyan.org/view/2251/94882) (译言网)
+* [持续集成（第二版）](http://article.yeeyan.org/view/2251/94882) (译言网)<!--需要必要电子邮件和微博账号，等待删除-->
 * [約耳談軟體(Joel on Software)](https://web.archive.org/web/20170615232349/http://local.joelonsoftware.com/wiki/首頁) (繁体中文)
```

Comment:
Don you forget this line? same as https://github.com/EbookFoundation/free-programming-books/pull/3166/files#r264002947

Reply:
The same as L250 , I will open a new PR to remove those books that is referenced to dead link.
In the Description of the PR, I had added a comment about it ^^.